app/src/controllers/submission.ts

```diff
@@ -462,6 +469,38 @@ const controller = {
     } catch (e: unknown) {
       next(e);
     }
+  },
+
+  /**
+   * @function assignPriority
+   * assigns a priority level to a submission based on given criteria
+   * criteria defined below
+   */
+  assignPriority: (submission: Partial<Submission>) => {
+    const matchesPriorityOneCriteria = // Priority 1 Criteria:
+      submission.singleFamilyUnits === '50-500' || // 1. More than 50 units of any type
+      submission.singleFamilyUnits === '>500' ||
+      submission.multiFamilyUnits === '50-500' ||
+      submission.multiFamilyUnits === '>500' ||
+      submission.otherUnits === '50-500' ||
+      submission.otherUnits === '>500' ||
+      submission.hasRentalUnits === 'Yes' || // 2. Supports Rental Units
+      submission.financiallySupportedBC === 'Yes' || // 3. Social Housing
+      submission.financiallySupportedIndigenous === 'Yes'; // 4. Indigenous Led
+
+    const matchesPriorityTwoCriteria = // Priority 2 Criteria:
+      submission.singleFamilyUnits === '10-49' || // 1. Single Family homes w/ >9 Units
```

Comment:
Values should reference defined enum types.

Reply:
created enums and changed to reference them.

Reply:
removed ones I created and added ones that we had